sale_order_line_date/__manifest__.py

```diff
@@ -4,6 +4,7 @@
 # © 2016 ForgeFlow, S.L.
 # Copyright 2017 Serpent Consulting Services Pvt. Ltd.
 # Copyright 2018 Camptocamp SA
+# Copyright 2023 Akretion
```

Comment:
You cannot do that, put yourself in CONTRIBUTORS.rst

Reply:
sorry, I remove it.